client/doc.go

```diff
@@ -16,7 +16,7 @@
 
 /*
 Package client and its KV API has been deprecated for external usage. Please use
-a postgres-compatible SQL driver (e.g. github.com/lib/pq). For more details, see
+a postgres-compatible SQL driver (e.g. github.com/cockroachdb/pq). For more details, see
```

Comment:
I think this should continue to recommend `lib/pq` instead of our fork; most clients would be better off sticking with the standard version instead of this modified version given its limitations. Use of `cockroachdb/pq` should be limited to the `cli` package.